js/src/Ice/Exception.js

```diff
@@ -2,87 +2,12 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
 
-const toString = function (key, object, objectTable, ident) {
```

Comment:
This was used to override the standard toString method and print all attributes. It seems overcomplicated and of little use.

Reply:
That's not a concern for local exceptions, since they must all provide a message. But it could be a concern for user exceptions that never provide any message.

When you print (toString) a user exception, what do you get? Only the type ID?
In C++ and C#, we give users the ability to define a custom "toString" for their user exceptions. How does this work in JS?

Reply:
In JS you can patch the class prototype to override a method. This would allow users to provide their own toString